highlighters/main/test-data/empty-line.zsh

```diff
@@ -31,6 +31,6 @@
 BUFFER=$'\\\n; ls'
 
 expected_region_highlight=(
-  '3 3 unknown-token' # ;
+  '3 3 commandseparator' # ;
```

Comment:
Unrelated: I wonder if we should add a style for the backslash-newline.

Reply:
I wouldn't be opposed, though it'd probably prompt allowing highlighting of all backslashes as well.